test/dca/strategies/Strategy.test.ts

```diff
@@ -57,6 +71,10 @@ export function testDcaStrategy(description: string, deployStrategy: Function, s
       await this.snapshot.restore()
     })
 
+    testStrategyDeposit()
+    testStrategyWithdraw()
+    testStrategyLimit()
+
     for (const strategySpecificTest of strategySpecificTests) {
       strategySpecificTest()
```

Comment:
`testConfigFile` argument is missing.

Reply:
forgot to remove the parameter requirement for strategySpecificTest, the config is passed through 'this'